src/cursor/cur_backup.c

```diff
@@ -207,22 +351,204 @@ __backup_log_append(WT_SESSION_IMPL *session, WT_CURSOR_BACKUP *cb, bool active)
     return (ret);
 }
 
+/*
+ * __backup_config --
+ *     Backup configuration.
+ */
+static int
+__backup_config(WT_SESSION_IMPL *session, WT_CURSOR_BACKUP *cb, const char *cfg[],
+  WT_CURSOR_BACKUP *othercb, bool *foundp, bool *log_only, bool *incr_only)
+{
+    WT_CONFIG targetconf;
+    WT_CONFIG_ITEM cval, k, v;
+    WT_CONNECTION_IMPL *conn;
+    WT_DECL_ITEM(tmp);
+    WT_DECL_RET;
+    const char *uri;
+    bool incremental_config, is_dup, log_config, target_list;
+
+    conn = S2C(session);
+    *foundp = *incr_only = *log_only = false;
```

Comment:
Can we initialize returned side-effects separately, before local variables?

Reply:
Fixed.